onnxruntime/server/grpc/grpc_app.cc

```diff
@@ -0,0 +1,29 @@
+#include "grpc_app.h"
+#include <grpcpp/health_check_service_interface.h>
+#include <grpcpp/ext/channelz_service_plugin.h>
+#include <grpcpp/ext/proto_server_reflection_plugin.h>
+namespace onnx_grpc = onnxruntime::server::grpc;
+
+namespace onnxruntime {
+namespace server {
+GRPCApp::GRPCApp(const std::shared_ptr<onnxruntime::server::ServerEnvironment>& env, std::string host, const unsigned short port) : m_service(env) {
+  m_server = std::unique_ptr<::grpc::Server>(nullptr);
```

Comment:
Why do you do this? Why not just do `make_unique` on line 21 instead of calling `swap`

Reply:
I'm stupid - I can just assign the output of BuildAndStart directly to m_server.